src/common/composables/amcharts5/tree-map-helper.ts

```diff
@@ -23,26 +21,29 @@ export const setTreemapTooltipText = (series: am5hierarchy.Treemap, tooltip: am5
     tooltip.label.setAll({
         fontSize: 14,
     });
-
-    const categoryFieldName = series.get('categoryField') || '';
     const valueFieldName = series.get('valueField') || '';
+    const colorFieldName = 'background_color';
 
     tooltip.label.adapters.add('text', (_, target) => {
+        const colorValue = target.dataItem?.dataContext?.[colorFieldName] || 'black';
         let value = target.dataItem?.dataContext?.[valueFieldName] || '-';
+
         if (currency) value = currencyMoneyFormatter(value, currency, currencyRate);
-        return `{${categoryFieldName}}: [bold]${value}[/] ({valuePercentTotal.formatNumber("0.00")}%)`;
+        return `[${colorValue}; fontSize: 10px]●[/] {category}: [bold]${value}[/] ({valuePercentTotal.formatNumber("0.00")}%)`;
```

Comment:
Just question:  what does `●` mean?

Reply:
It's dot in tooltip :)
![스크린샷 2022-12-27 오후 6 51 03](https://user-images.githubusercontent.com/18563857/209648227-3ae81099-d04f-455e-9ddf-dc04cddb9922.png)

Reply:
![image](https://user-images.githubusercontent.com/19162140/209648602-26574af4-937b-4455-8b1f-35dbb67c2a80.png)
show this shape